src/main/webapp/package.json

```diff
@@ -76,7 +76,7 @@
     "select2": "^3.5.1",
     "showdown": "^1.7.4",
     "sortablejs": "^1.5.1",
-    "styled-components": "^4.3.2",
+    "styled-components": "^5.0.0",
```

Comment:
Which pages will this affect? I see line list, but which others?

Reply:
Cart and Linelist are the main ones.